themes/finna2/js/js-helper.js

```diff
@@ -0,0 +1,250 @@
+/* exported jsHelper */
```

Comment:
Osassa funktioita käytetään while-luuppia ja osassa for-luuppia tekemään käytännössä sama asia. Voisi yhtenäistää käyttämään toista kattavammin.